.github/workflows/rust-ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
           submodules: recursive
       - name: Install protobuf-compiler
         run: sudo apt-get install -y protobuf-compiler
+      - name: Install SDL2
```

Comment:
would it be harmful to just append this to the step above and change the name?

Reply:
I just wanted to make it line up with the README, where we treat these as individual packages.  If we had more than half a dozen of these, then it would make sense to start appending them.

Reply:
If you feel strongly about this, then I will make the change,

Reply:
I made the change.  Please take a look.